.github/workflows/stale-action.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   stale:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/stale@98ed4cb500039dbcccf4bd9bedada4d0187f2757 # renovate: tag=v3.0.19
+      - uses: actions/stale@cdf15f641adb27a71842045a94023bef6945e3aa # renovate: tag=v4.0.0
```

Comment:
I've checked if we need to make any changes:

> ##### Breaking Changes
> 
>     * The options `skip-stale-issue-message` and `skip-stale-pr-message` were removed. Instead, setting the options `stale-issue-message` and `stale-pr-message` will be enough to let the stale workflow add a comment. If the options are unset, a comment will not be added which was the equivalent of setting `skip-stale-issue-message` to `true`.
> 
>     * The `operations-per-run` option will be more effective. After migrating, you could face a failed-fast process workflow if you let the default value (30) or set it to a small number. In that case, you will see a warning at the end of the logs (if enabled) indicating that the workflow was stopped sooner to avoid consuming too much API calls. In most cases, you can just increase this limit to make sure to process everything in a single run.

We don't have any `skip-stale-issue-message` or `skip-stale-pr-message` options, so this one does not apply to us.

We do set our own `operations-per-run` limit, which will be more effective in the new major version. The current limit is set to `40`. I've checked the logs of the last run of the `stale` action:

```
Statistics:
Processed items: 883
├── Processed issues: 832
└── Processed PRs   : 51
Fetched items: 883
Operations performed: 20
```

So we're not exceeding any limits now, but we might with the new behavior. In any case, if the action complains we can just raise the limit in small steps until it stops complaining. 😄 

I'll let the Renovate team do their own review now. 😉